core/MY_Model.php

```diff
@@ -1984,7 +1984,7 @@ public function __call($method, $arguments)
         $parent_class = get_parent_class($this);
         if ($parent_class !== FALSE && !method_exists($parent_class, $method) && !method_exists($this,$method))
         {
-            echo 'No method with that name ('.$method.') in MY_Model or CI_Model.';
+            echo 'No method with that name ('.$method.') in '. get_class($this) .' or MY_Model or CI_Model.';
```

Comment:
can you add `exit;` in next line? so no need to proceed the execution after reach this.